akka-http-handler/src/main/scala/com/softwaremill/sttp/akkahttp/AkkaHttpHandler.scala

```diff
@@ -35,10 +39,19 @@ class AkkaHttpHandler private (actorSystem: ActorSystem,
 
   override def send[T](r: Request[T, S]): Future[Response[T]] = {
     implicit val ec: ExecutionContext = this.ec
+
+    val connectionSettings = ClientConnectionSettings(actorSystem)
```

Comment:
Maybe it would make sense to store the "base" client/connection pool settings in the class, so as not to parse the config on each request? And then specialize in the `send` method?